README.md

```diff
@@ -317,24 +317,54 @@ Deploy [web validate](https://github.com/retaildevcrews/webvalidate) to drive co
 
 ```bash
 
+# Add Log Analytics extension
+az extension add -n log-analytics
+
+# create Log Analytics for the webv clients
+az monitor log-analytics workspace create -g $He_App_RG -l $He_Location -n $He_Name -o table
+
+# retrieve the Log Analytics ID
+export He_LogAnalyticsId=$(az monitor log-analytics workspace show -g $He_App_RG -n $He_Name --query customerId -o tsv)
+
+# add Log Analytics Keys to Key Vault (TODO: should secondary keys be added?)
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "LogAnalyticsPrimaryKey" --value $(az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query primarySharedKey -o tsv)
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "LogAnalyticsSecondaryKey" --value $(az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query secondarySharedKey -o tsv)
+
+# get the values
+export He_LogAnalytics_Primary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsPrimaryKey)
+export He_LogAnalytics_Secondary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsSecondaryKey)
```

Comment:
this stores both keys in clear text in an env var and then writes them in clear text to disk

the env var should be set to the command in 330-331 and then $(eval $He_*)

by using eval, if the env vars are compromised, you can't get the actual value unless you have permissions.